straxen/misc.py

```diff
@@ -7,10 +7,17 @@
 import warnings
 import datetime
 import pytz
+from sys import getsizeof, stderr
+from itertools import chain
+from collections import OrderedDict, deque
 from importlib import import_module
 from git import Repo, InvalidGitRepositoryError
 from configparser import NoSectionError
 import typing as ty
+try:
+    from reprlib import repr
+except ImportError:
+    pass
```

Comment:
This is actually not needed, its just an optimization. `repr` is a bultin so if you dont have reprlib it wont raise an error when you try to use `repr`, it will just use the builtin implementation.
I dont think its important, i just copied this snippet from python recipes, perhaps the person who wrote the snippet also worked on reprlib so they included the attempt to import it if it exists.
This is a similar pattern to the 
```python
try:
    import cPickle as pickle
except ImportError:
    pass

that people would always add add the top of their scripts/packages before cPickle became a builtin and the default.